### ISSUE TITLE
integrating private fastq files

### ISSUE THREAD

**Issue by kfuku52** [kfuku52/amalgkit#27]
Is there any streamlined way to integrate your own fastq files not derived from SRA? If not, I'll implement a subcommand for it but I assume there is, as you work on both public and private data. Please let me know.

**Comment by Hego-CCTB** [comment]
I'm having trouble with this myself, right now. Currently, there is no good way. You have to create your own metadata, count & quant files manually. Basically making amalgkit believe it's working on public data. 

Once the metadata is in place, the rest should be easily integratable into `amalgkit`.  I wonder if we can auto-generate metadata based off user fastq data. All that's needed for `amalgkit` to work is: `species`,`tissue` and `run` (i.e. a unique ID). On the other hand, since `species` and `tissue` require user interaction anyways, it may just be mor efficient to just make a Wiki page on how to create your own metadata and don't bother with automisation. What do you think?


`quant` in it's current implementation retrieves metadata directly from SRA, which will likely cause problems (either because of throwing an error, and/or because we lose the fastp info we just worked on getting back into metadata) , but that should be an easy fix.

One last issue I predict has to do with how R operates in cases where you want to mix private data with SRA data. Because number of columns will not match between the custom metadata and the SRA metadata, R will certainly complain when trying to integrate both into the same dataframe.

**Comment by kfuku52** [comment]
I manually prepared a metadata table for my specific use last week. 

I guess the best way would be to automatically produce a boilerplate tsv from available fastq files, maybe as a part of `amalgkit sanity`, and navigate users to manually fill out the table. Wiki will greatly help.

**Comment by Hego-CCTB** [comment]
I'll make a separate issue for `amalgkit sanity`, as we have collected a few tasks now that could be handled by that funcitonality.

**Comment by Hego-CCTB** [comment]
continued in https://github.com/kfuku52/amalgkit/issues/54
